tests/backup/backup_helper.go

```diff
@@ -1960,3 +1960,65 @@ func CreateNamespaceLabelScheduleBackupWithoutCheck(scheduleName string, cluster
 	}
 	return resp, nil
 }
+
+// namespaceLabelBackupSuccessCheck verifies if the labeled namespaces are backed up and checks for labels applied to backups
+func namespaceLabelBackupSuccessCheck(backupName string, ctx context.Context, listOfLabelledNamespaces []string, namespaceLabel string) error {
```

Comment:
function Name should start with Capital, change it to NamespaceLabelBackupSuccessCheck